src/Components/Cart/Cart.js

```diff
@@ -1,15 +1,41 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './Cart.scss';
 
-const Cart = ({ cartItems }) => {
-  console.log(cartItems)
+const Cart = () => {
+  
+  const [data, setData] = useState([]);
 
-  const itemsInCart = cartItems.map((product) => {
+  useEffect(() => {
+    const fetchData = async (api) => {
+      const response = await fetch(api)
+      const responseJson = await response.json()
+      setData(responseJson)
+    }
+    fetchData('http://localhost:3001/api/v1/cart')
+  }, [])
+
+
+  const [cartTotal, setCartTotal] = useState('');
+
+  useEffect(() => {
+    const calculateSubTotal = () => {
+      const subTotal = data.reduce((sum, item) => {
+        sum += (item.quantity * parseInt(item.price));
+        return sum;
+      }, 0)
+      setCartTotal(subTotal)
+    }
+    calculateSubTotal()
+  }, [data])
+
```

Comment:
Great, a function for subtotal. How are we going to calculate the tax?

Reply:
I pulled down and I'm working on the tax and total as we speak!